php/class-coauthors-wp-list-table.php

```diff
@@ -276,8 +283,8 @@ function extra_tablenav( $which ) {
 			echo '<option value="' . esc_attr( $key ) . '" ' . selected( $this->active_filter, $key, false ) . '>' . esc_attr( $value ) . '</option>';
 		}
 		echo '</select>';
+	    submit_button( __( 'Filter', 'co-authors-plus' ), 'secondary', false, false );
```

Comment:
I moved the `submit_button()` into the `if ( ! empty( $this->filters ) )` block to ensure that the button is not visible in case there are no filters available, e.g. in case a client tried to overwrite/remove the filters.